sdk/resourcemanager/Proto.Client/billing/Azure.ResourceManager.Billing/autorest.md

```diff
@@ -0,0 +1,13 @@
+# Generated code configuration
```

Comment:
This file should be removed similar to authorization, you will need to manually set the tag which pulls in the autorest folder via the link